test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -798,6 +798,16 @@ def forward(self, input):
         model = CumSum()
         self.run_test(model, x)
 
+    def test_meshgrid(self):
+        class Meshgrid(torch.nn.Module):
+            def forward(self, x, y, z):
+                return torch.meshgrid(x, y, z)
+
+        x = torch.ones(3, requires_grad=True)
```

Comment:
could you use torch.rand instead of torch.ones and torch.zeros. It could capture some potential failure cases that are not covered with torch.ones/torch.zeros.